.gitignore

```diff
@@ -1 +1,3 @@
-.vscode/
\ No newline at end of file
+.vscode/
+.DS_Store
+assets/.DS_Store
```

Comment:
I think `**/.DS_Store` will work for any directory here instead of targeting specific directory.